core/include/gnuradio-4.0/PluginLoader.hpp

```diff
@@ -140,6 +140,21 @@ class PluginLoader {
     BlockRegistry           *_registry;
     std::vector<std::string> _knownBlocks;
 
+    gr_plugin_base *
+    handlerForName(std::string_view name) const {
+        auto it = _handlerForName.find(std::string(name));
+        if (it == _handlerForName.end()) {
+            fmt::print("Error: No handler for name {}, available ", name);
```

Comment:
API: should either be an `std::expected<>` (preferred, if this is used more often), throw a `gr::exception(..)`, but not be a print-out. This won't be visible and/or actionable by the developer/user.

Reply:
Removed output. The function already returns nullptr on error,  wrapping in expected would not bring much to the table.